README.md

```diff
@@ -342,61 +346,54 @@ Please, make sure to install the following software/packages on the VMs.
 - Node v16.17.1 (only _host_)
 
   ```bash
-  curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
-  nvm install 16.17.1
+  curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash && nvm install 16.17.1
   ```
 
 - Npm (only _host_)
 
   ```bash
-  sudo apt update
-  sudo apt install nodejs npm
+  sudo apt update --yes && sudo apt install --yes npm
   ```
 
 - Python 2 (only _host_)
 
   ```bash
-  sudo apt install python2 && alias python="/usr/bin/python2
+  sudo apt install python2 --yes && alias python="/usr/bin/python2"
   ```
 
-- Solc v0.5.16 (only _host_)
-
-  ```bash
-  sudo snap install solc
-  ```
+- Solc (solidity compiler) and Ganache CLI (only _host_)
 
-- Ganache CLI (only _host_)
   ```bash
-  sudo npm install -g ganache
+  npm install --global solc ganache
```

Comment:
I think so. Let me just retry to run the setup on a cleaned VM to check.

Reply:
I can confirm it doesn't work. I think I had installed `solc` with `snap` and I saw that you could install `solc` with `npm` so I modified the documentation. But it doesn't work because `npm` actually installs [`solcjs`](https://www.npmjs.com/package/solc) which is different from `solc`. Sorry for the inconvenience.